src/types/video.ts

```diff
@@ -281,11 +281,13 @@ export interface ReactVideoRenderLoaderProps {
   resizeMode?: EnumValues<VideoResizeMode>;
 }
 
-export interface ReactVideoProps extends ReactVideoEvents, ViewProps {
+export interface ReactVideoProps
+  extends ReactVideoEvents,
+    Omit<ViewProps, 'style'> {
   source?: ReactVideoSource;
   /** @deprecated Use source.drm */
   drm?: Drm;
-  style?: StyleProp<ViewStyle>;
+  style?: StyleProp<ViewStyle | CSSProperties>;
```

Comment:
This change was [suggested in the original PR](https://github.com/TheWidlarzGroup/react-native-video/pull/4333#issuecomment-2614437684), but causes an tsc error in the native `Video.tsx` component, since `View` doesn't accept `React.CSSProperties` to `style`.

There are at least two options:

1. Remove this style here, and require users to force the value of `style` to `any` or something else when using `style` on web with web-only styles, since the types in `Video.web.tsx` are not exposed as part of the public API conditional to the platform.
2. Force the type of `style` internally in `Video.tsx`, to satisfy `tsc`, which could technically be considered incorrect since `React.CSSProperties` are not supported by `View` but the types wouldn't indicate that.

Reply:
Since using `unstable_createElement`, you don't need to specify this `| CSSProperties` since `ViewStyle` will automatically be converted to css.

`ViewStyle` already contains some web-only props, if someone wants to use some css props not supported on `ViewStyle` they need to use `any` (like with any other RN component)

Reply:
Makes sense. I've removed the changes to this file.